pom.xml

```diff
@@ -182,16 +183,30 @@
             <scope>test</scope>
         </dependency>
 
+        <!-- For GlobalMatrixAuthorizationStrategy -->
```

Comment:
Usually you would instead use `MockAuthorizationStrategy`.

Reply:
Good to know, but I'm not sure I want to change that functionality.  I'm trying to keep the changes to a minimum.  @KostyaSha thoughts?

Reply:
Fine, just FYI.